test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -17,3 +20,185 @@ Describe "Windows aliases do not conflict with Linux commands" -Tags "CI" {
+            "curl -> Invoke-WebRequest"               = $FullCLR
+            "cvpa -> Convert-Path"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "dbp -> Disable-PSBreakpoint"             = $FullCLR -or $CoreWindows -or $CoreUnix
+            "del -> Remove-Item"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "diff -> Compare-Object"                  = $FullCLR -or $CoreWindows
+            "dir -> Get-ChildItem"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "dnsn -> Disconnect-PSSession"            = $FullCLR -or $CoreWindows -or $CoreUnix
+            "ebp -> Enable-PSBreakpoint"              = $FullCLR -or $CoreWindows -or $CoreUnix
+            "echo -> Write-Output"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "epal -> Export-Alias"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "epcsv -> Export-Csv"                     = $FullCLR -or $CoreWindows -or $CoreUnix
+            "epsn -> Export-PSSession"                = $FullCLR
+            "erase -> Remove-Item"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "etsn -> Enter-PSSession"                 = $FullCLR -or $CoreWindows -or $CoreUnix
+            "exsn -> Exit-PSSession"                  = $FullCLR -or $CoreWindows -or $CoreUnix
+            "fc -> Format-Custom"                     = $FullCLR -or $CoreWindows -or $CoreUnix
+            "fhx -> Format-Hex"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "fl -> Format-List"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "foreach -> ForEach-Object"               = $FullCLR -or $CoreWindows -or $CoreUnix
+            "ft -> Format-Table"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "fw -> Format-Wide"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gal -> Get-Alias"                        = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gbp -> Get-PSBreakpoint"                 = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gc -> Get-Content"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gcb -> Get-Clipboard"                    = $FullCLR
+            "gci -> Get-ChildItem"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gcm -> Get-Command"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gcs -> Get-PSCallStack"                  = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gdr -> Get-PSDrive"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "ghy -> Get-History"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gi -> Get-Item"                          = $FullCLR -or $CoreWindows -or $CoreUnix
+            "gin -> Get-ComputerInfo"                 = $CoreWindows -or $CoreUnix
```

Comment:
This looks so much better, but you can use a little more whitespace will help quickly scanning for differences.
```powershell
"gin -> Get-ComputerInfo"                 =              $CoreWindows -or $CoreUnix
```
There are a few more lines below that you could do something similar.